### PR TITLE
reflect the EOM of Symfony 2.3

### PR DESCRIPTION
**Pull request by xabbuh**

### CODE REVIEW

File: components/form/introduction.rst [hunk 1]
```diff
@@ -163,14 +163,14 @@ component offers a rich integration.
 
 To use the integration, you'll need the ``TwigBridge``, which provides integration
 between Twig and several Symfony components. If you're using Composer, you
-could install the latest 2.3 version by adding the following ``require``
+could install the latest 2.7 version by adding the following ``require``
```
Comment (xabbuh) [inline components/form/introduction.rst]: The installation instructions in this document should probably be rewritten, but that should be done in a different PR.

**Comment by javiereguiluz** [comment]
👍

File: components/form/introduction.rst (hunk 1 above)
Comment (xabbuh) [inline components/form/introduction.rst]: see #6639
